docs/querying-data.rst

```diff
@@ -0,0 +1,196 @@
+
+Use one of the ``collection_id`` returned by the previous query to filter your results to a single collection. To get a summary of a whole contracting process rather choose a ``compiled_release`` or ``record`` collection.
+
+The following query returns a top-level summary of the first 3 contracting processes in collection ``1259``.
+
+.. code-block:: sql
+
+  select
+      *
+  from
+      release_summary
+  where
+      collection_id = 1259
+  limit 3;
+
+To learn more about the summaries and aggregates in the ``release_summary`` table, refer to the :ref:`release_summary` documentation.
+
+To get data from a different collection, change the ``collection_id`` parameter.
+
+Calculate the total value of tenders in a collection
+----------------------------------------------------
+
+Summary data on tenders is stored in the ``tender_summary`` table.
+
+The following query calculates the total value of tenders disaggregated by currency and tender status in collection ``1259``.
+
+.. code-block:: sql
+
+  select
+    tender_value_currency, --return the currency of the tender value, values in OCDS have an amount and a currency, as datasets may contain values in multiple currencies
+    tender_status,
+    sum(tender_value_amount)
+  from
+    tender_summary
+  where
+    collection_id = 1259
+  group by
+    tender_value_currency,
+    tender_status
+  order by
+    tender_value_currency,
+    tender_status;
+
+To learn more about the summaries and aggregates in the ``tender_summary`` table, refer to the :ref:`tender_summary` documentation.
+
+.. tip::
+
+  The ``tender``, ``award`` and ``contract`` objects in OCDS all have a ``.status`` property.
+
+  Kingfisher Views exposes these status properties in ``tender_summary.tender_status``, ``awards_summary.award_status`` and ``contracts_summary.contract_status``.
+
+  Consider which statuses you want to include or exclude from your analysis, for example you might wish to exclude pending and cancelled contracts when calculating the total value of contracts for each buyer.
+
+  The `OCDS codelist documentation <https://standard.open-contracting.org/latest/en/schema/codelists/#>`__ describes the meaning of the statuses for each object.
+
+Calculate the top 10 buyers by award value
+------------------------------------------
+
+Summary data on buyers is stored in the ``buyer_summary`` table and summary data on awards is stored in the ``award_summary`` table.
+
+To join summary tables, use the ``id`` column, which uniquely identifies a release. To learn more about the relationships between tables refer to the :ref:`relationships` documentation.
+
+The ``buyer_summary`` table doesn't include the buyer's name, however the ``buyer`` column contains a jsonb blob of the buyer  for each contracting process, which this can be extracted from.
+
+Most summary tables in Kingfisher Views include a column containing jsonb blobs of the object the summary relates to.
```

Comment:
Have we standardised these yet? I remember there being variants such as `data` and `release` etc. If we've standardised them it might be worth shouting out what they tend to be called and giving an example e.g. `release_summary_with_data.data`

Reply:
Not yet https://github.com/open-contracting/kingfisher-views/issues/132

Reply:
Should we update now that #132 is closed?

Reply:
Updated in https://github.com/open-contracting/kingfisher-views/pull/109/commits/bd0e98e59adb874c29eabbe69e4881790d7a698a